internal/service/streamconnection/resource_stream_connection_test.go

```diff
@@ -235,6 +235,62 @@ func TestAccStreamPrivatelinkEndpoint_streamConnection(t *testing.T) {
 	})
 }
 
+func TestAccStreamRSStreamConnection_AWSLambda(t *testing.T) {
+	var (
+		resourceName   = "mongodbatlas_stream_connection.test"
+		projectID      = os.Getenv("MONGODB_ATLAS_ASP_PROJECT_EAR_PE_ID") //test-acc-tf-p-keep-ear-AWS-private-endpoint project has aws integration
+		instanceName   = acc.RandomName()
+		connectionName = acc.RandomName()
+		roleArn        = os.Getenv("MONGODB_ATLAS_ASP_PROJECT_AWS_ROLE_ARN")
+	)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             CheckDestroyStreamConnection,
+		Steps: []resource.TestStep{
+			{
+				Config: awsLambdaStreamConnectionConfig(projectID, instanceName, connectionName, roleArn),
+				Check:  awsLambdaStreamConnectionAttributeChecks(resourceName, instanceName, connectionName, roleArn),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportStateIdFunc: checkStreamConnectionImportStateIDFunc(resourceName),
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccStreamRSStreamConnection_AWSLambdaForLocal(t *testing.T) {
```

Comment:
I also added a local acc testing. Let me know if we can remove it if the above test is working in CI flow. Thanks